pavement.py

```diff
@@ -120,7 +120,7 @@ def compile_file(fullname, ddir=None, force=0, rx=None, quiet=0):
             if not force:
                 try:
                     mtime = int(os.stat(fullname).st_mtime)
-                    expect = struct.pack('<4sl', imp.get_magic(), mtime)
+                    expect = struct.pack('<4sl', MAGIC_NUMBER, mtime)
```

Comment:
thanks for the update... the compile_file code shoudl be removed ... and just use upstream code


We had this "fork" here to support py2 migration... and I forgot to clean the code